content/en/synthetics/settings/_index.md

```diff
@@ -23,9 +23,7 @@ On the [Synthetic Monitoring Settings page][1], you can adjust the following set
 
 * [Global Variables](#global-variables)
 * [Private Locations][2]
-* [Default settings](#default-settings)
-  * [Default Locations](#default-locations)
-  * [APM integration for Browser Tests](#apm-integration-for-browser-tests)
```

Comment:
why remove that?

Reply:
Because this child topic display appears at the beginning of the page and replicates the On This Page menu on the right, I think it's alright to have the three general Settings tabs instead of the general tabs and the sub-tabs.